stubs/tensorflow/tensorflow/strings.pyi

```diff
@@ -0,0 +1,241 @@
+    name: str | None = None,
+) -> tuple[RaggedTensor, RaggedTensor]: ...
+@overload
+def unicode_encode(
+    input: TensorCompatible,
+    output_encoding: Literal["UTF-8", "UTF-16-BE", "UTF-32-BE"],
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    name: str | None = None,
+) -> Tensor: ...
+@overload
+def unicode_encode(
+    input: RaggedTensor,
+    output_encoding: Literal["UTF-8", "UTF-16-BE", "UTF-32-BE"],
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    name: str | None = None,
+) -> RaggedTensor: ...
+@overload
+def unicode_script(input: TensorCompatible, name: str | None = None) -> Tensor: ...
+@overload
+def unicode_script(input: RaggedTensor, name: str | None = None) -> RaggedTensor: ...
+@overload
+def unicode_split(
+    input: StringTensorCompatible,
+    input_encoding: str,
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    name: str | None = None,
+) -> Tensor | RaggedTensor: ...
+@overload
+def unicode_split(
+    input: RaggedTensor,
+    input_encoding: str,
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    name: str | None = None,
+) -> RaggedTensor: ...
+@overload
+def unicode_split_with_offsets(
+    input: StringTensorCompatible,
+    input_encoding: str,
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    name: str | None = None,
+) -> tuple[_TensorOrRaggedTensor, _TensorOrRaggedTensor]: ...
+@overload
+def unicode_split_with_offsets(
+    input: RaggedTensor,
+    input_encoding: str,
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    name: str | None = None,
+) -> tuple[RaggedTensor, RaggedTensor]: ...
+@overload
+def unicode_transcode(
+    input: StringTensorCompatible,
+    input_encoding: str,
+    output_encoding: Literal["UTF-8", "UTF-16-BE", "UTF-32-BE"],
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    replace_control_characters: bool = False,
+    name: str | None = None,
+) -> Tensor: ...
+@overload
+def unicode_transcode(
+    input: RaggedTensor,
+    input_encoding: str,
+    output_encoding: Literal["UTF-8", "UTF-16-BE", "UTF-32-BE"],
+    errors: Literal["replace", "strict", "ignore"] = "replace",
+    replacement_char: int = 65533,
+    replace_control_characters: bool = False,
+    name: str | None = None,
+) -> RaggedTensor: ...
+def unsorted_segment_join(
+    inputs: StringTensorCompatible,
+    segment_ids: UIntTensorCompatible,
+    num_segments: UIntTensorCompatible,
+    separator: str = "",
+    name: str | None = None,
+) -> Tensor: ...
+@overload
+def upper(input: TensorCompatible, encoding: Literal["utf-8", "", " "] = "", name: str | None = None) -> Tensor: ...
```

Comment:
Again the docstring doesn't say `" "` is allowed.

Reply:
Fixed in ae35a8b3e.